components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__genes.xml

```diff
@@ -137,10 +137,14 @@
         var infoTool = new Element('span', {'class' : 'fa fa-info-circle xHelpButton gene-info', 'title' : geneNames[i]});
         var icon_helpController = new PhenoTips.widgets.HelpButton(infoTool);
         var triangle = new Element('span', {'class': "fa fa-exclamation-triangle", title: '$services.localization.render('phenotips.patientSheetCode.termSuggest.nonStandardGenesymbol')'}).update(' ');
-        if (isSuggestSelected)
-          element.insert({after: infoTool});
-        else
+        if (isSuggestSelected) {
+          if (!$('inline')) {
```

Comment:
Just FYI, there's `XWiki.contextaction`  (which can be `view`, `edit`, `inline`, `admin`) available in the JS API. No need the modify the code though, this is also fine.